synapse/lib/jupyter.py

```diff
@@ -38,6 +38,7 @@ def getDocPath(fn, root=None):
     Raises:
         ValueError if the file does not exist or directory traversal attempted..
     '''
+    s_common.deprecated('synapse.lib.jupyter.getDocPath', curv='2.175.0', eolv='3.0.0')
```

Comment:
We will need a changelog entry for the deprecation of the `synapse.lib.jupyter` path.

Do we want to discuss a earlier eolv or eolv date so we can prune it sooner?

Reply:
I consider that an "internal API" that we are free to carve out whenever we would like

Reply:
I've seen the jupyter helpers used by third parties ~3 years ago; but cannot speak to them being currently used by anyone.  I don't want to keep them around any longer than needed though; and given the deprecation notices that'll be embedded in any output, they'd be producing bad looking output anyhow.